accounts/views.py

```diff
@@ -803,6 +805,18 @@ def handle_uploaded_file(user_id, f):
     return True
 
 
+def copy_sound_file_to_mirror_location(source_path):
```

Comment:
If you want to leave it like this, OK, but to me it seems like only a little work to make a start on this by creating a `sound_mirror` or `sound_upload` util file